drivers/sensor/amg88xx/Kconfig

```diff
@@ -0,0 +1,106 @@
+# Kconfig - AMG88XX infrared thermopile sensor configuration options
+
+#
+# Copyright (c) 2017 Phytec Messtechnik GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig AMG88XX
+	bool
+	prompt "AMG88XX Infrared Thermopile Sensor"
+	depends on I2C
+	default n
+	help
+	  Enable driver for AMG88XX infrared thermopile sensor.
+
+config AMG88XX_NAME
+	string
+	prompt "Driver name"
+	default "AMG88XX"
+	depends on AMG88XX
+	help
+	  Device name with which the AMG88XX sensor is identified.
+
+config AMG88XX_I2C_ADDR
+	hex
+	prompt "I2C address for TMP006 Sensor"
+	depends on AMG88XX
+	default "0x68"
+	help
+	  I2C address of the AMG88XX sensor.
+
+	  0x68: AD-SELECT connected GND
+	  0x69: AD-SELECT connected VDD
+
+config AMG88XX_I2C_MASTER_DEV_NAME
+	string
+	prompt "I2C master where AMG88XX is connected"
+	depends on AMG88XX
+	default "I2C_0"
+	help
+	  Specify the device name of the I2C master device to which the
+	  AMG88XX chip is connected.
+
+choice
+	prompt "Trigger mode"
+	depends on AMG88XX
+	default AMG88XX_TRIGGER_NONE
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config AMG88XX_TRIGGER_NONE
+	bool
+	prompt "No trigger"
+
+config AMG88XX_TRIGGER_GLOBAL_THREAD
+	bool
+	prompt "Use global thread"
+	depends on GPIO
+	select AMG88XX_TRIGGER
+
+config AMG88XX_TRIGGER_OWN_THREAD
+	bool
+	prompt "Use own thread"
+	depends on GPIO
+	select AMG88XX_TRIGGER
+
+endchoice
+
+config AMG88XX_TRIGGER
+	bool
+	depends on AMG88XX
+
+config AMG88XX_GPIO_DEV_NAME
+	string
+	prompt "GPIO device"
+	default "GPIO_0"
+	depends on AMG88XX && AMG88XX_TRIGGER
+	help
+	  The device name of the GPIO device to which the AMG88XX interrupt
+	  (alert) pin is connected.
+
+config AMG88XX_GPIO_PIN_NUM
+	int
+	prompt "Interrupt GPIO pin number"
+	default 0
+	depends on AMG88XX && AMG88XX_TRIGGER
+	help
+	  The number of the GPIO pin on which the interrupt signal from the
```

Comment:
How about:

     The GPIO pin number receiving the interrupt signal from the AMG88XX sensor.